src/main/java/uk/gov/hmcts/reform/laubackend/idam/utils/InputParamsVerifierHelper.java

```diff
@@ -86,4 +86,19 @@ public static void verifyPositiveNumeric(String value, String message) throws In
             }
         }
     }
+
+    public static void verifyPositiveNumericAndNotNull(String value, String message) throws InvalidRequestException {
+        if (isEmpty(value)) {
```

Comment:
You can combine first 2 if/else/if statements into one. if (isEmpty(value) || !isNumeric(value))

Reply:
This will cause the issue for third check , when we are checking number is less than MAX positive number so leaving as it is